src/models/User/Model.ts

```diff
@@ -67,7 +67,7 @@ export class User extends Model<User> {
 
   public async getCompany() {
     const companyUser = await this.getCompanyUser();
-    if (companyUser !== undefined) return Company.findByPk(companyUser.companyUuid);
+    if (companyUser) return Company.findByPk(companyUser.companyUuid);
     return;
```

Comment:
creo que este return esta al pedo

Reply:
No, si lo sacas te tira algo asi: `TS7030: Not all code paths return a value.`